tests/orm/data/test_code.py

```diff
@@ -0,0 +1,48 @@
+"""Tests for the `Code` class."""
+# pylint: disable=redefined-outer-name
+import pytest
+
+from aiida.orm import Code
+
+
+@pytest.fixture
+def create_codes(tmpdir, aiida_localhost):
+    """Create a local and remote `Code` instance."""
+    import os
+
+    filename = 'add.sh'
+    filepath = os.path.join(tmpdir.realpath(), filename)  # Python 3.5 does not support Path objects for `join` yet
+
+    with open(filepath, 'w'):
+        pass
+
+    code_local = Code(local_executable=filename, files=[filepath]).store()
+    code_remote = Code(remote_computer_exec=(aiida_localhost, '/bin/true')).store()
+
+    return code_local, code_remote
+
+
+@pytest.mark.usefixtures('clear_database_before_test')
+def test_get_full_text_info(create_codes):
+    """Test the `Code.get_full_text_info` method."""
+    for code in create_codes:
+        full_text_info = code.get_full_text_info()
+
+        assert isinstance(full_text_info, list)
+        assert ['PK', code.pk] in full_text_info
```

Comment:
Since the result is a list of lists with consistent sorting, wouldn't it be simpler to just test

```python
assert full_text_info == [['PK', code.pk], ['UUID', code.uuid], ...]
```

Reply:
Ah never mind, I realized it's because of the local / remote distinction below.